src/client/Killboard.tsx

```diff
@@ -74,76 +103,17 @@ export class KillboardFull extends Killboard {
             <th>Kills</th>
             <th>Deaths</th>
           </tr>
-          <tr>
-            <td>
-              <img src={goldQueen} />
-            </td>
-            <td>{this.state[Character.GoldQueen].kills}</td>
-            <td>{this.state[Character.GoldQueen].deaths}</td>
-          </tr>
-          <tr>
-            <td>
-              <img src={goldStripes} />
-            </td>
-            <td>{this.state[Character.GoldStripes].kills}</td>
-            <td>{this.state[Character.GoldStripes].deaths}</td>
-          </tr>
-          <tr>
-            <td>
-              <img src={goldAbs} />
-            </td>
-            <td>{this.state[Character.GoldAbs].kills}</td>
-            <td>{this.state[Character.GoldAbs].deaths}</td>
-          </tr>
-          <tr>
-            <td>
-              <img src={goldSkulls} />
-            </td>
-            <td>{this.state[Character.GoldSkulls].kills}</td>
-            <td>{this.state[Character.GoldSkulls].deaths}</td>
-          </tr>
-          <tr>
-            <td>
-              <img src={goldChecks} />
-            </td>
-            <td>{this.state[Character.GoldChecks].kills}</td>
-            <td>{this.state[Character.GoldChecks].deaths}</td>
-          </tr>
-          <tr>
-            <td>
-              <img src={blueQueen} />
-            </td>
-            <td>{this.state[Character.BlueQueen].kills}</td>
-            <td>{this.state[Character.BlueQueen].deaths}</td>
-          </tr>
-          <tr>
-            <td>
-              <img src={blueStripes} />
-            </td>
-            <td>{this.state[Character.BlueStripes].kills}</td>
-            <td>{this.state[Character.BlueStripes].deaths}</td>
-          </tr>
-          <tr>
-            <td>
-              <img src={blueAbs} />
-            </td>
-            <td>{this.state[Character.BlueAbs].kills}</td>
-            <td>{this.state[Character.BlueAbs].deaths}</td>
-          </tr>
-          <tr>
-            <td>
-              <img src={blueSkulls} />
-            </td>
-            <td>{this.state[Character.BlueSkulls].kills}</td>
-            <td>{this.state[Character.BlueSkulls].deaths}</td>
-          </tr>
-          <tr>
-            <td>
-              <img src={blueChecks} />
-            </td>
-            <td>{this.state[Character.BlueChecks].kills}</td>
-            <td>{this.state[Character.BlueChecks].deaths}</td>
-          </tr>
+          <KillboardRow {...{stat: this.state[Character.GoldQueen], image: goldQueen}}/>
```

Comment:
Is this the same as this?

```ts
<KillboardRow stat={this.state[Character.GoldQueen]} image={goldQueen} />
```

Reply:
yes.. i can change that.  
I guess your way is the more standard react way so ill do it.

Reply:
No worries, I'll take care of all the minor stuff like that :)